lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -2745,6 +2762,9 @@ class SPIRVAtomicInstBase : public SPIRVInstTemplateBase {
     if (getOpCode() == OpAtomicStore &&
         getOperand(ValueOperandIndex)->getType()->isTypeInt(64))
       Module->addCapability(CapabilityInt64Atomics);
+    if (getOpCode() == OpAtomicFAddEXT)
```

Comment:
There is a comment above this method that says that it exists only because of `OpAtomicStore` - this is not true anymore, please update the comment as well